tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -266,6 +281,22 @@ def _test_storage_type(storage_type, fsx_fs_id, region):
         assert_that(get_storage_type(fsx_fs_id, region)).is_equal_to("SSD")
 
 
+def get_deployment_type(fsx_fs_id, region):
+    logging.info("Getting DeploymentType from DescribeFilesystem API.")
+    fsx = boto3.client("fsx", region_name=region)
+    return (
+        fsx.describe_file_systems(FileSystemIds=[fsx_fs_id])
+        .get("FileSystems")[0]
+        .get("LustreConfiguration")
+        .get("DeploymentType")
+    )
+
+
+def _test_deployment_type(deployment_type, fsx_fs_id, region):
+    # The default deployment type is SCRATCH_1
```

Comment:
I don't think it's always true. 
In china AFAIK the only deployment_type supported in China regions is PERSISTENT_1.

If we don't know what deployment_type will be used, because empty, we can skip the check.

Reply:
Done, but as I know, we only test deployment type in commercial region, and the default type is SCRATCH_1, should we test it?